packages/components/tests/dummy/app/templates/components/badge-count.hbs

```diff
@@ -0,0 +1,118 @@
+{{page-title "BadgeCount component"}}
+
+<h2 class="dummy-h2">BadgeCount</h2>
+
+<section>
+  <h3 class="dummy-h3" id="bc-overview"><a href="#bc-overview" class="dummy-link-section">§</a> Overview</h3>
+  <p class="dummy-paragraph"><code class="dummy-code">BadgeCount</code>
+    is a numeric label used to display things like version number and collection enumeration in tabs.</p>
+  <p class="dummy-paragraph"><em>Notice: don’t use
+      <code class="dummy-code">BadgeCount</code>
+      for anything non-numeric, except version numbers (ie. “v1.2.0”).</em></p>
+</section>
+
+<section>
+  <h3 class="dummy-h3" id="bc-component-api"><a href="#bc-component-api" class="dummy-link-section">§</a>
+    Component API</h3>
+  <p class="dummy-paragraph" id="component-api-badge-count">Here is the API for the component:</p>
+  <dl class="dummy-component-props" aria-labelledby="component-api-badge-count">
+    <dt>size <code>enum</code></dt>
+    <dd>
+      <p>Acceptable values:</p>
+      <ol>
+        <li>small</li>
+        <li class="default">medium</li>
+        <li>large</li>
+      </ol>
+    </dd>
+    <dt>type <code>enum</code></dt>
+    <dd>
+      <p>Acceptable values:</p>
+      <ol>
+        <li class="default">filled</li>
+        <li>inverted</li>
+        <li>outlined</li>
+      </ol>
+    </dd>
+    <dt>color <code>enum</code></dt>
+    <dd>
+      <p>Acceptable values:</p>
+      <ol>
+        <li class="default">neutral</li>
+        <li>neutral-dark-mode</li>
+      </ol>
+    </dd>
+    <dt>text <code>string</code></dt>
+    <dd>
+      <p>The text value that should go in the badge counter.</p>
+    </dd>
+    <dt>...attributes</dt>
+    <dd>
+      <p><code class="dummy-code">...attributes</code> spreading is supported on this component.</p>
+    </dd>
+  </dl>
+</section>
+
+<section>
+  <h3 class="dummy-h3" id="bc-how-to-use"><a href="#bc-how-to-use" class="dummy-link-section">§</a> How to use</h3>
+  <p class="dummy-paragraph">Invocation of the component would look something like this:</p>
+  {{! prettier-ignore-start }}
+  <CodeBlock
+    @language="markup"
+    @code='
+      <Hds::BadgeCount @text="3" @size="large" @type="inverted" />
+    '
+  />
+  {{! prettier-ignore-end }}
+</section>
+<section>
+  <h3 class="dummy-h3" id="accessibility"><a href="#accessibility" class="dummy-link-section">§</a> Accessibility</h3>
+  <p class="dummy-paragraph">This component has been designed and implemented with accessibility in mind. When used as
+    recommended, there should not be any accessibility issues with this component.
+  </p>
+</section>
+
+<section data-test-percy>
```

Comment:
now that you have duplicated the pages, you also have to duplicate the visual regression tests.
in the file `packages/components/tests/acceptance/percy-test.js` at line `32-33` you see the test for the `badge` route/page; you have to add (just duplicate the lines) a test for the `badge-count` route